.circleci/config.yml

```diff
@@ -15,31 +15,31 @@ aliases:
         - master
 
 orbs:
-  codecov: codecov/codecov@1.0.5
-  docker: circleci/docker@2.1.1
+  codecov: codecov/codecov@4.0.1
+  docker: circleci/docker@2.5.0
 
 executors:
   linux-executor: # this executor is for general testing and packaging linux binaries
     working_directory: ~/go/src/github.com/klaytn/klaytn
     resource_class: medium
     docker:
-      - image: klaytn/build_base:1.2-ubuntu18.04-go.1.20.6-solc0.8.13
+      - image: klaytn/build_base:1.12-ubuntu20.04-go.1.20.6-solc0.8.13
```

Comment:
Name of docker image go version is go1.20.6. Shouldn't it be 1.22.1?

Reply:
yes, maybe I missed it, I will fix now